config/config.yaml.dist

```diff
@@ -55,7 +55,7 @@ cadence:
     host: ""
 #    port: 7933
 #    domain: "pipeline"
-#    createNonexistentDomain: false
+#    createNonexistentDomain: true
```

Comment:
**Required**: we **MUST** set this back to false, it is overridden in the installer, my apologies this was my mistake.